docs/source/checks/tabular/model_evaluation/plot_roc_report.py

```diff
@@ -17,12 +18,15 @@
 What is the ROC Report check?
 ==============================
 The ``ROCReport`` check calculates the ROC curve for each class.
-
+The ROC curve is a plot of TPR (true positive rate) with respect to FPR (false positive rate)
+at various thresholds.
+Thr ROC curve is one of many tools for checking model performance or compare multiple
+models.
```

Comment:
This sentence doesn't really add any additional info about this metric
I'd either drop it or add a link to more info about the ROC from the deepchecks glossary:
https://deepchecks.com/glossary/roc-receiver-operating-characteristic-curve/